src/midcom/datamanager/template/view.php

```diff
@@ -79,7 +79,7 @@ public function form_label(FormView $view, array $data)
         if (!$data['label']) {
             $data['label'] = $this->renderer->humanize($data['name']);
```

Comment:
the `humanize` call here should be removed, since you do it again two lines further down